src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/UserSyncService.kt

```diff
@@ -67,16 +62,16 @@ data class AuthUser(
   @Schema(description = "User Name in Auth", example = "Global Search User", required = true)
   val username: String,
 
-  @Schema(description = "email", example = "jimauth@justice.gov.uk", required = true)
-  val email: String?
+  @Schema(description = "email", example = "jimauth@justice.gov.uk", required = false)
+  val email: String = ""
```

Comment:
We're making the field optional - could we leave it as `String?` because if it's not provided then it's null?

Reply:
Yes - I think you're right.  I didn't quite like the error message from MapDifference.
It shows: 
 **not equal: only on left={email=user1nomis@digital.justice.gov.uk}"**
 rather than my preferred:
**value differences={email=(user1nomis@digital.justice.gov.uk, )}**
But looking back I think the original does make sense....the email only exists on the left - which is fine.
I thought it was a bit confusing when you have a totally missing entry, but as long as it's interpreted correctly then it should be ok.
Will update